### ISSUE TITLE
Not able to add via scan

### ISSUE THREAD

**Issue by preraksola** [Sneezry/authenticator#53]
Hello,

I created an extension using your code. But the scanning feature does not work on some web pages. It says, "cannot capture. Please refresh the page".
But at the same time, if I use your extension, it works. 
Any idea what may be leading to this behavior?

My extension is not been published on the Chrome Store yet. Can that be an issue?

Thanks!

Prerak

**Comment by Sneezry** [comment]
Hi @preraksola ,

You need give a response in content script. Listen 'capture' action, and callback 'beginCapture'. See the code at https://github.com/Sneezry/authenticator/blob/master/javascript/content.js#L107

**Comment by preraksola** [comment]
I am already doing that, but on some tabs it is not returning `beginCapture`. On Friday for a particular tab, it was showing the error but today for the same tab, it's working.

That's a bit weird!

**Comment by Sneezry** [comment]
Hi @preraksola ,

Does it work fine today? You need reload all tabs after you load your extension, so that content script can be injected :)

**Comment by preraksola** [comment]
Yeah, I tried that too. On some machine it's working but on some it's still showing the error to reload the page. Unfortunately I am not able to reproduce that behavior on my machine but it's showing the error on my colleague's machine.

**Comment by preraksola** [comment]
Hi @Sneezry ,

I found the method to reproduce the error on each machine.

- Download this repository
- In `chrome://extensions/` turn on the `Developer Mode`.
- Click `Pack Extension`. In the popup window, select the downloaded folder `authenticator-master`. Leave blank for the certificate file if you don't have it.
- This will generate the `.crx` file. Drag and drop this file in `chrome://extensions/` tab and click `Add Extension`.
- Now open any QR code file in Chrome and try to add it in the extension by scanning it.

What I have seen so far is, if the URL of the tab is starting with `file:///` it is throwing the error. 
I also added the sample image in a HTML file and opened that file in the browser, but still it showed the same message. (Here also, the URL start with `file:///`)
But, if I add that file on my local Apache server and open it via localhost, it is able to scan it. (Here the URL starts with `localhost/`).

The behavior is same, if I download your extension from the Chrome web store.
So, is there anything that can be done to resolve this?
